ckanext/depositar_theme/templates/home/index/idx_showcases.html

```diff
@@ -0,0 +1,39 @@
+<div class="idx showcases">
+  <div class="container">
+    <h1>{{ _('Showcase') }}</h1>
+    <div id="recipeCarousel" class="carousel slide w-100" data-ride="carousel">
+      <a class="carousel-control-prev w-auto" href="#recipeCarousel" role="button" data-slide="prev">
+        <i class="fa fa-chevron-left"></i>
+      </a>
+      <div class="carousel-inner" role="listbox">
+        {% for case in h.get_showcase() %}
+          {% if case == h.get_showcase()[0] %}
+            <div class="carousel-item active">
+          {% else %}
+            <div class="carousel-item">
+          {% endif %}
+            <div class="card-col col-md-4">
+              <div class="card card-body">
+                <div class="img-container">
+                  <img class="img" src={{ case.image_url }}>
+                </div>
+                <div class="content">
+                  <h3 class="case-title">{{ case.title }}</h3>
+                  <div class="case-content">
+                    <div>{{ case.content }}</div>
+                  </div>
+                  <a class="btn more" href="{{ h.url_for('/showcase/' + case.href) }}">
```

Comment:
改為 Flask endpoint (route) name
route list 可以在 Flask-DebugToolbar (debug = True 時的右上角工具) -> Route List 查詢

```suggestion
                  <a class="btn more" href="{{ h.url_for('showcase_blueprint.read', id=case.href) }}">
```